src/library/scala/Console.scala

```diff
@@ -122,7 +122,7 @@ import scala.util.DynamicVariable
  *             a body of code. Threadsafe by virtue of [[scala.util.DynamicVariable]].
  *
  */
-object Console extends DeprecatedConsole with AnsiColor {
+object Console extends AnsiColor {
```

Comment:
Ideally, we'd also not inherit AnsiColor, as it's in `scala.io`, which shouldn't be a requirement of the core.